.changeset/fresh-cheetahs-pump.md

```diff
@@ -0,0 +1,5 @@
+---
+"@azure-tools/cadl-ranch-specs": major
```

Comment:
Regenerate the changeset. I don't think this be major. Should be just a patch.

Reply:
Fixed in the new version.